modules/thrift/src/thrift/concurrency/Mutex.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <thrift/concurrency/Mutex.h>
+
+namespace apache
+{
+namespace thrift
+{
+namespace concurrency
+{
```

Comment:
This and several other files are mainly only pulled in and slightly modified as a workaround for missing `std::thread` and `std::mutex` support in Zephyr. I would prefer to avoid unnecessary code churn on them if possible.

Reply:
Fair enough for a temporary solution/workaround.